README.md

```diff
@@ -1,38 +1,5 @@
 # EAP 5G
 wpa_supplicant modified version for EAP 5G
```

Comment:
```suggestion
# TNGFUE
wpa_supplicant modified version for EAP 5G
```

Reply:
and move the line `wpa_supplicant modified version for EAP 5G` in top of `wpa_supplicant/READMD`